### ISSUE TITLE
Accessibility Parity

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1128]
After Lighthouse reports were run and analyzed the following items need to be addressed to get us closer to parity for accessibility. These have already been technically assessed and should be VERY minor development efforts, but please let me know any concern: 

1. For the header elements, we need to add the correct ARIA labels that are there in Revize so screen readers can properly announce the functionality present. 
See screenshot

<img width="1027" alt="Image" src="https://github.com/user-attachments/assets/d5fc9939-98b9-4f3d-abd7-0b4ed0a5ddbf" />

Report link to see this present (validate that this is not being flagged on revize pages): https://pagespeed.web.dev/analysis/https-main--clarkcountynv--aemsites-aem-live-government-assessor-property_search/2lqrke0206?form_factor=desktop

Since this is present in the header, this hurts the accessibility score on every page

2. Cards need an update so that list items are not flagged for accessibility. These cards are not list items on revize today
See screenshot

<img width="1010" alt="Image" src="https://github.com/user-attachments/assets/d047b1ef-2198-40b7-8069-44c71435d99a" />
Report link to see this present (validate that this is not being flagged on revize pages): https://pagespeed.web.dev/analysis/https-main--clarkcountynv--aemsites-aem-live-government-assessor-property_search/2lqrke0206?form_factor=desktop

Since this is happening on these card displays, this issue is present on many of the highest viewed pages

**Comment by meejain** [comment]
@sarah0arnold  - The above ask for a major refactor of the blocks. However we have been trying improving upon the Accessibility stats via

**Comment by sarah0arnold** [comment]
I understand your feedback ^ and I don't know that this is critical for launch, but it does need to be completed. @meejain  can you estimate how long it would take to complete? Thank you!
FYI @PatrickClarkCounty I believe this is in reference to issue #2 logged in the original ticket.
